user_scripts/check_run_stats.sh

```diff
@@ -1,3 +1,9 @@
+########################################################
+# README
+# this file uses variant_calling.yaml for its parameters
```

Comment:
Keeping variant_calling and check_run_stats with the same yaml file makes sense - variant_calling depends on the outputs of check_run_stats to only perform variant calling on the haplotypes that mapped to correct locations of genome